items.json

```diff
@@ -93240,14 +93240,14 @@
     ]
   },
   {
-    "id": "crpg_knights_fall_v1_h0",
-    "baseId": "crpg_knights_fall_v1",
+    "id": "crpg_knights_fall_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.